test/Air/AirParser.test.js

```diff
@@ -1418,6 +1418,20 @@ describe('#AirParser', () => {
         });
     });
 
+    it('should parse booking with warnings and missing fare info and should show messages', () => {
+      const uParser = new Parser('universal:UniversalRecordImportRsp', 'v36_0', { });
+      const parseFunction = airParser.AIR_CREATE_RESERVATION_REQUEST;
+      const xml = fs.readFileSync(`${xmlFolder}/UniversalRecordWithWarnings.xml`).toString();
+      return uParser.parse(xml)
+        .then(json => parseFunction.call(uParser, json))
+        .then((result) => {
+          expect(result[0].fareQuotes.length).to.be.eq(0);
+          expect(result[0].messages.length).to.be.eq(2);
+          expect(result[0].messages[0]._).to.be.eq('Reservation is currently being updated by another process.');
```

Comment:
Is it going to be an array in case when only one message returned? Cause parser used under the hood could collapse things. I would probably add one more test for the case with one message

Reply:
it won't https://github.com/Travelport-Ukraine/uapi-json/blob/master/src/Request/default-config.js#L19